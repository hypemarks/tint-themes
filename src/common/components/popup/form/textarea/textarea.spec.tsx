import { render } from '@testing-library/react';
import React from 'react';
import TextArea from './textarea';

describe('Test <TextArea /> Component', () => {
  it('should render snapshot', () => {
    const wrapper = render(<TextArea value={'value'} name={'name'} onChange={jest.fn} id={'id'} onBlur={jest.fn} />);
    expect(wrapper.asFragment()).toMatchSnapshot();
  });
});
