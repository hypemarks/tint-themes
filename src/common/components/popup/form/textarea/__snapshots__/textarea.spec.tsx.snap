// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test <TextArea /> Component should render snapshot 1`] = `
<DocumentFragment>
  <textarea
    class="undefined textarea"
    id="id"
    name="name"
    placeholder="Type here..."
  >
    value
  </textarea>
</DocumentFragment>
`;
